packages/components/src/components/Snackbar/useSnackbar.tsx

```diff
@@ -25,8 +25,10 @@ export function useSnackbar() {
   const show = (message: string, snackbarOptions?: SnackbarOptions) => {
     const { offset, setOffset } = context
 
-    // Adjust offset if provided or reset to default
-    const newOffset = snackbarOptions?.offset || defaultOffset
+    const customOffset = snackbarOptions?.offset
+
+    // Custom offset if provided, else default
+    const newOffset = customOffset === undefined ? defaultOffset : customOffset
```

Comment:
Fixing bug with offset `0` setting the offset to 60 (defaultOffset) due to 0 being falsy.